packages/text/src/components/Caption.js

```diff
@@ -13,8 +12,8 @@ export default class Caption extends React.Component {
 
   static propTypes = {
     children: PropTypes.node.isRequired,
-    size: PropTypes.oneOf(Object.keys(fontSizes)),
-    type: PropTypes.oneOf(Object.keys(fontTypes)),
+    size: PropTypes.oneOf(Object.values(fontSizes)),
```

Comment:
I think we can leave this property type flexible, and use the fontSizes just as suggestion